tox.ini

```diff
@@ -10,6 +10,7 @@ setenv =
 deps =
        -r{toxinidir}/requirements.txt
        -r{toxinidir}/test-requirements.txt
+       .[yaml,toml,baseline]
```

Comment:
If doing it here, you can make this simpler with

```suggestion
extras = yaml toml baseline
```

Reply:
https://tox.wiki/en/latest/config.html#extras for additional context and docs

Reply:
Yeah, I tried this originally, but pep8 would still fail. But maybe cause I had commas?

Reply:
Apparently, the pep8 job was skipping installing of the package, therefore would skip the extras. So I removed the `skip_install`
https://github.com/tox-dev/tox/issues/1105#issuecomment-448602810